pkg/cli/bootstrap.go

```diff
@@ -254,65 +227,15 @@ func HookInUrfaveCLI(ctx context.Context, cancel context.CancelFunc, a *cli.App,
 	}
 }
 
-// urfaveBefore is a cli.BeforeFunc that implements tracing and automatic updating
-//nolint:funlen // Why: Not worth splitting out yet. May want to do so w/ more CLI support.
-func urfaveBefore(a *cli.App, logger logrus.FieldLogger, exit func(), cleanup *func(),
-	exitCode *int) func(c *cli.Context) error {
+// urfaveBefore is a cli.BeforeFunc that implements tracing
+func urfaveBefore(a *cli.App) func(c *cli.Context) error {
 	return func(c *cli.Context) error {
-		cargs := c.Args().Slice()
-		command := ""
-		args := make([]string, 0)
-		if len(cargs) > 0 {
-			command = cargs[0]
-		}
-		if len(cargs) > 1 {
-			args = cargs[1:]
-		}
-
 		trace.AddInfo(c.Context, log.F{
-			a.Name + ".subcommand": command,
-			a.Name + ".args":       strings.Join(args, " "),
+			a.Name + ".subcommand": c.Args().First(),
```

Comment:
Can we do a consistent `cli.subcommand` it's kinda PITA in telemetry. And since you're here, I don't have to :)

Also add usage, so we have a full command but without arguments, doing metrics over array is not great.

Reply:
Yeah, fine w/ me.